### ISSUE TITLE
Is this still being developed? 

### ISSUE THREAD

**Issue by BenMitchell1979** [gbowerman/azure-minecraft#1]
I noticed that the scriptcraft deploy is failing because it depends on Ubuntu 15.04 - this build is not available any longer on Azure. It has been replaced by 16.04-LTS. I was going to update some of the files and wanted to verify if the file locations the Shell Scripts calls on are still valid.

**Comment by gbowerman** [comment]
@BenMitchell1979 sorry I only just noticed the issue. As you probably surmised, the scriptcraft template isn't being maintained. I just made a few changes to update the Azure API and Ubuntu OS versions, plus make it used Managed Disks instead of unmanaged - https://github.com/gbowerman/azure-minecraft/commit/464d1d6ae7192b16f460c4fe4104d54b5fbb3114. Not tested it though. I'm not sure if there's a later version of the server out there.. Thanks for bringing it to my attention, and apologies again that my attention was slow to notice.
